package.json

```diff
@@ -42,53 +42,53 @@
     "@radix-ui/react-slot": "1.1.1",
     "@radix-ui/react-tabs": "1.1.1",
     "@radix-ui/react-toast": "1.2.5",
-    "@react-router/express": "7.6.0",
-    "@react-router/fs-routes": "7.6.0",
-    "@react-router/node": "7.6.0",
+    "@react-router/express": "7.6.2",
+    "@react-router/fs-routes": "7.6.2",
+    "@react-router/node": "7.6.2",
     "@scure/bip32": "1.6.2",
     "@scure/bip39": "1.5.4",
-    "@supabase/supabase-js": "2.49.8",
+    "@supabase/supabase-js": "2.50.0",
```

Comment:
unrelated upgrade. I did it because I thought maybe newer version will make supabase realtime connect faster